debian/control

```diff
@@ -0,0 +1,18 @@
+Source: ice-setup
+Maintainer: Dan Mick <dan.mick@redhat.com>
+Section: python
+Priority: optional
+Build-Depends: python-setuptools (>= 0.6b3), python-all (>= 2.6.6-3), debhelper (>= 7.4.3)
+Standards-Version: 3.9.1
+X-Python-Version: >=2.6
```

Comment:
Yeah.  that substitutes in from debian/ice-setup.substvars which is created magically by dh_python2